go-controller/pkg/allocator/pod/macs.go

```diff
@@ -0,0 +1,166 @@
+package pod
+
+import (
+	"fmt"
+	"net"
+
+	kubevirtv1 "kubevirt.io/api/core/v1"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/klog/v2"
+	k8snet "k8s.io/utils/net"
+
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/kubevirt"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/types"
+	"github.com/ovn-org/ovn-kubernetes/go-controller/pkg/util"
+)
+
+// getMACOwner compose the owner identifier reserved for MAC addresses management.
+// Returns "<ns>/<pod-name>" for regular pods and "<ns>/<vm-name>" for VMs.
+func getMACOwner(pod *corev1.Pod) string {
```

Comment:
```suggestion
// macOwner compose the owner identifier reserved for MAC addresses management.
// Returns "<ns>/<pod-name>" for regular pods and "<ns>/<vm-name>" for VMs.
func macOwner(pod *corev1.Pod) string {
```

nit: no point prefixing this with get. This is suggested in effective go: https://go.dev/doc/effective_go#Getters

Reply:
Done